scripts/build-glean-metadata.py

```diff
@@ -183,23 +189,29 @@ def etl_snake_case(line: str) -> str:
 
             metric_type = metric.definition["type"]
             metric_name_snakecase = stringcase.snakecase(metric.identifier)
-            metric_table_name = (
-                f"{metric.bq_prefix}.{metric_name_snakecase}"
-                if metric.bq_prefix
-                else f"metrics.{metric_type}.{metric_name_snakecase}"
-            )
-            bigquery_names = dict(
+            etl = dict(
                 stable_ping_table_names=stable_ping_table_names,
-                metric_type=metric_type,
-                metric_table_name=metric_table_name,
-                glam_etl_name=etl_snake_case(metric.identifier),
+                table_name=(
+                    f"{metric.bq_prefix}.{metric_name_snakecase}"
+                    if metric.bq_prefix
+                    else f"metrics.{metric_type}.{metric_name_snakecase}"
+                ),
             )
+            if metric_type != "event" and GLAM_PRODUCT_MAPPINGS.get(app.app_id):
+                (glam_product, glam_app_id) = GLAM_PRODUCT_MAPPINGS[app.app_id]
+                glam_metric_id = etl_snake_case(metric.identifier)
+                etl["glam_url"] = (
+                    "https://glam.telemetry.mozilla.org/"
+                    + f"{glam_product}/probe/{glam_metric_id}/explore"
+                    + f"?app_id={glam_app_id}"
+                )
```

Comment:
Writing it this way partly to appease flake8, though I think it might be a little easier to read in any case.